Dockerfile

```diff
@@ -1,44 +1,42 @@
 ARG BASE=11.5.0-devel-ubuntu20.04
 FROM nvidia/cuda:${BASE}
 
-LABEL maintainer="J.C.Knight@sussex.ac.uk"
-LABEL version="4.8.0"
-LABEL org.opencontainers.image.documentation="https://genn-team.github.io/"
-LABEL org.opencontainers.image.source="https://github.com/genn-team/genn"
-LABEL org.opencontainers.image.title="GeNN Docker image"
+LABEL maintainer="J.C.Knight@sussex.ac.uk" \
```

Comment:
thank you for the tidying here - I didn't realise just how directly dockerfile lines mapped to layers and thus massive disk wastage when I wrote this